platform/src/main/java/org/stellar/anchor/platform/config/PlatformApiConfig.java

```diff
@@ -35,7 +35,7 @@ public boolean supports(@NotNull Class<?> clazz) {
   @Override
   public void validate(@NotNull Object target, @NotNull Errors errors) {
```

Comment:
Can we add an extra validation here: if network set to mainnet, throw an error if auth type is set to NONE?
This will prevent unsecured endpoints in production/staging envs

Reply:
@Ifropc I am not sure if we should add the validation here since the platform server is now separated from the sep server. We should be careful because this disables the anchor to have open endpoints in their private network. If you think this is necessary, let's create a separate thread in Slack to discuss.

Reply:
That's a good point. I think we should just print WARN when auth set to NONE instead